python/GafferSceneUI/RenderPassEditor.py

```diff
@@ -544,6 +548,11 @@ def __renameSelectedRenderPass( self ) :
 		renderPassName = dialogue.waitForRenderPassName( parentWindow = self.ancestor( GafferUI.Window ) )
 		if renderPassName is not None and renderPassName != selectedRenderPasses[0] :
 
+			nonEditableReason = GafferScene.EditScopeAlgo.renameRenderPassNonEditableReason( editScope, renderPassName )
+			if nonEditableReason != "" :
```

Comment:
This would be `nonEditableReason is not None` if we took my suggestion for the bindings.